index.html

```diff
@@ -20,15 +20,16 @@
       <div class="header-background">
         <div class="container">
           <div class="row">
-            <div class="column-full">
+            <div class="column-full header-content">
               <h1 class="header">Code Journal</h1>
+              <a class="entries-anchor">Entries</a>
             </div>
           </div>
         </div>
       </div>
     </header>
     <main class="container">
-      <div data-view="entry-form">
+      <div id="entry-form" data-view="entry-form" class="not-hidden">
```

Comment:
What is this class accomplishing? It doesn't look like you're applying any styles to this are you?